### PR TITLE
[stable/elasticsearch-exporter] Add support for PodSecurityPolicies, ServiceAccounts

### PR DESCRIPTION
**Pull request by ptran32**
This PR adds support for PodSecurityPolicies and ServiceAccounts.

They are not activated by default in order to avoid potentially breaking changes, and the default service account is the one used by default for the same reason.

Signed-off-by: Patrice Tran <ptran@mailjet.com>

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X]  Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/chart]`)

/assign @svenmueller

### CODE REVIEW

File: stable/elasticsearch-exporter/values.yaml [hunk 1]
```diff
@@ -159,3 +159,15 @@ prometheusRule:
     #   annotations:
     #     description: The heap usage is over 90% for 15m
     #     summary: ElasticSearch node {{$labels.node}} heap usage is high
+
+# Create a service account by default
```
Comment (svenmueller) [inline stable/elasticsearch-exporter/values.yaml]: i think this comment is a bit misleading. Maybe we can drop "by default"?

**Comment by svenmueller** [comment]
@ptran32 please have a look at the failing checks

File: stable/elasticsearch-exporter/values.yaml (hunk 1 above)
Comment (ptran32) [reply stable/elasticsearch-exporter/values.yaml]: I've removed "by default" as suggested.

**Comment by ptran32** [comment]
> @ptran32 please have a look at the failing checks

I've fixed the failing checks

**Pull request merged**